riff-raff/app/views/snippets/recordTable.scala.html

```diff
@@ -17,6 +17,7 @@
         <th>Recipe</th>
         <th>Hosts</th>
         <th>Status</th>
+        <th>Time taken</th>
```

Comment:
Can I suggest `Duration` rather than `Time taken`? Would save the table header going to two rows.